pipelines/pingcap/tiflow/latest/pull_cdc_integration_pulsar_test.groovy

```diff
@@ -0,0 +1,170 @@
+                    retry(2) {
+                        sh label: "download third_party", script: """
+                            cd ../tiflow && ./scripts/download-integration-test-binaries.sh ${REFS.base_ref} && ls -alh ./bin
+                            make check_third_party_binary
+                            cd - && mkdir -p bin && mv ../tiflow/bin/* ./bin/
+                            ls -alh ./bin
+                            ./bin/tidb-server -V
+                            ./bin/pd-server -V
+                            ./bin/tikv-server -V
+                            ./bin/tiflash --version
+                            ./bin/sync_diff_inspector --version
+                        """
+                    }
+                }
+                dir("tiflow") {
+                    cache(path: "./bin", filter: '**/*', key: prow.getCacheKey('binary', REFS, 'cdc-integration-test')) { 
+                        // build cdc, pulsar_consumer, cdc.test for integration test
+                        // only build binarys if not exist, use the cached binarys if exist
+                        sh label: "prepare", script: """
+                            ls -alh ./bin
+                            [ -f ./bin/cdc ] || make cdc
+                            [ -f ./bin/cdc_pulsar_consumer ] || make pulsar_consumer
+                            [ -f ./bin/cdc.test ] || make integration_test_build
+                            ls -alh ./bin
+                            ./bin/cdc version
+                        """
+                    }
+                    cache(path: "./", filter: '**/*', key: "ws/${BUILD_TAG}/tiflow-cdc") { 
+                        sh label: "prepare", script: """
+                            cp -r ../third_party_download/bin/* ./bin/
+                            ls -alh ./bin
+                        """
+                    }
+                }
+            }
+        }
+
+        stage('Tests') {
+            when { expression { !skipRemainingStages} }
+            matrix {
+                axes {
+                    axis {
+                        name 'TEST_GROUP'
+                        values 'G00', 'G01', 'G02', 'G03', 'G04', 'G05', 'G06',  'G07', 'G08', 'G09',
+                            'G10', 'G11', 'G12', 'G13', 'G14', 'G15', 'G16', 'G17'
+                    }
+                }
+                agent{
+                    kubernetes {
+                        namespace K8S_NAMESPACE
+                        yamlFile POD_TEMPLATE_FILE
+                        defaultContainer 'golang'
+                    }
+                } 
+                stages {
+                    stage("Test") {
+                        options { timeout(time: 40, unit: 'MINUTES') }
+                        environment { 
+                            TICDC_CODECOV_TOKEN = credentials('codecov-token-tiflow') 
+                            TICDC_COVERALLS_TOKEN = credentials('coveralls-token-tiflow')    
+                        }
```

Comment:
Suggest deleting these lines if the following test does not use these two variables.